src/Sulu/Bundle/AdminBundle/Resources/js/components/ColumnList/ToolbarDropdownList.js

```diff
@@ -20,20 +21,12 @@ export default class ToolbarDropdownList extends React.Component<Props> {
 
         return options.map((dropdownOptionConfig: ToolbarDropdownOptionConfig, columnIndex: number) => {
             const key = `option-${columnIndex}`;
-            const handleClick = () => {
-                dropdownOptionConfig.onClick(this.props.columnIndex);
-            };
+            const {onClick, label} = dropdownOptionConfig;
 
             return (
-                <li
-                    className={toolbarDropdownStyles.option}
-                    key={key}
-                    value={this.props.columnIndex}
```

Comment:
@trickreich Had to change this file, because the newer version of the no bind rule seems to recognize the inline `handleClick` function here. I realized that there is this `value` on `li`, this shouldn't have any impact, right?